README.md

```diff
@@ -104,6 +108,47 @@ at runtime as an environment variable:
 
 See `giftless/config.py` for some default configuration options. 
 
+
+### Running a local example
+
+First, create a new project on Github or any other platform.
+Here, we create `example-proj-datahub-io`. Add any data file
+to it. The goal is to track this possible large file with 
+git-lfs and use Giftless as the local server. In our example,
+we create a CSV named `research_data_factors.csv`.
+
+Next, start the Giftless server. Go to your local project 
+directory and run:
+
+```bash
+example-proj-datahub-io$ git init
+example-proj-datahub-io$ git remote add origin YOUR_REMOTE_REPO
+example-proj-datahub-io$ git lfs track '*.csv'
+example-proj-datahub-io$ git lfs track 
+example-proj-datahub-io$ git add .gitattributes #you should have a .gitattributes file at this point
+example-proj-datahub-io$ git add "*.csv"
+example-proj-datahub-io$ git commit -m "Tracking data files"
+example-proj-datahub-io$ git lfs ls-files # you should see your file listed
+example-proj-datahub-io$ git config -f .lfsconfig lfs.url http://127.0.0.1:5000/<user_or_org>/<repo>/
+# in our case, we used http://127.0.0.1:5000/datopian/example-proj-datahub-io/; 
```

Comment:
Please note that this repo is entirely unrelated to `datahub` or any other Datopian specific project and we should probably refrain from referencing anything specific in examples to avoid confusion.

Reply:
It's better to use something like `myorg/myproject` for example

Reply:
Sorry, I don't think `myorg/myproject` helps an entirely new developer. If the person only has their own Gitub account, it may not be clear what 'myorg' is. But surely I could use another string that is not 'datopian'. Note that I'm not using any reference to datahub though.